api-extractor/src/definitions/ApiDocumentation.ts

```diff
@@ -129,6 +129,22 @@ export default class ApiDocumentation {
   public parameters: { [name: string]: IParam; };
 
   /**
+   * A list of link elements to be processed after all basic documentation has been created 
+   * for all items in the project. We save the processing for later because we need ApiTag 
+   * information before we can deem a link element is valid.
+   * Example: If API item A has a link in it's documentation to API item B, then B must not
+   * have ApiTag.Internal. 
+   */
+  public links: ICodeLinkElement[];
+
+  /**
+   * A list of 'Tokens' that have been recognized as inheritdoc tokens that will be processed 
+   * after the basic documentation for all API items is complete. We save the processing for after 
+   * because we need ApiTag information before we can deem an inheritdoc token as valid.
+   */
+  public inheritdocs: Token[];
```

Comment:
Maybe `incompleteInheritdocs`, `inheritdocsToComplete` etc.  I'd also recommend to set this to undefined or empty after you're done with it.